src/js/components/Drop/StyledDrop.js

```diff
@@ -27,6 +27,97 @@ const dropKeyFrames = keyframes`
   }
 `;
 
+const isObject = object => object != null && typeof object === 'object';
+
+const deepEqual = (object1, object2) => {
+  const keys1 = Object.keys(object1);
+  const keys2 = Object.keys(object2);
+
+  if (keys1.length !== keys2.length) {
+    return false;
+  }
+
+  for (let i = 0; i < keys1.length; i += 1) {
+    const val1 = object1[keys1[i]];
+    const val2 = object2[keys1[i]];
+    const areObjects = isObject(val1) && isObject(val2);
+    if (
+      (areObjects && !deepEqual(val1, val2)) ||
+      (!areObjects && val1 !== val2)
+    ) {
+      return false;
+    }
+  }
+  return true;
+};
+
+const marginStyle = (theme, align, data, responsive, marginProp) => {
+  const margin = theme.global.edgeSize[data] || data;
+  // if user provides CSS string such as '50px 12px', apply that always
+  let customCSS;
+  let adjustedMargin;
+  if (typeof margin === 'string') {
+    customCSS = margin.split(' ').length > 1;
+  }
+  if (
```

Comment:
Wondering if a code comment here would be helpful? Maybe something along the lines of "// desired margin may need to be adjusted depending the drop's alignment"? Or do you think it is self evident?

Reply:
I had the comment up above I moved it down though I think its better

Reply:
I didn't see this come through in the commits, but may be missing. All else looks great. Nice work. I will approve now, but feel free to add the comment if you think it helps.